packages/swirl-components/src/components/swirl-table-row-group/swirl-table-row-group.tsx

```diff
@@ -13,13 +13,40 @@ export class SwirlTableRowGroup {
 
   @Prop() label!: string;
   @Prop() tooltip?: string;
+  @Prop() collapsable?: boolean = false;
+
+  @State() isVisible: boolean = true;
+
+  private toggleShowItems = () => {
+    const rowsContainer = this.el.shadowRoot.querySelector(
+      ".rows-container"
+    ) as HTMLElement;
+    if (this.isVisible) {
+      rowsContainer.style.height = "0";
```

Comment:
Issue: We need to hide the collapsed contents from the accessibility tree. You could use `display: none;` or `aria-hidden="true"` for example.